psi/bld/prepare-PSI.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
I see you're creating a new 'root' directory, psi. These dirs need special considerations to ensure they do not show up in the pax version. I advise to make psi a subdir of the smpe directory, which is still logically correct since you plan on using SMPE as base for PSI.

Reply:
ok I will make it under smpe folder